src/rules/package-jest-scripts-conform.ts

```diff
@@ -0,0 +1,68 @@
+import { buildRule } from './build-rule';
+import { PackageManifestSchema, RuleName } from './types';
+import type { RuleExecutionFailure } from '../execute-rules';
+
+export default buildRule({
+  name: RuleName.PackageJestScriptsConform,
+  description: 'Do the jest-related `scripts` in `package.json` conform?',
+  dependencies: [RuleName.RequireValidPackageManifest],
+  execute: async ({ project, template, pass, fail }) => {
+    const entryPath = 'package.json';
+
+    const templateManifest = await template.fs.readJsonFileAs(
+      entryPath,
+      PackageManifestSchema,
+    );
+
+    const projectManifest = await project.fs.readJsonFileAs(
+      entryPath,
+      PackageManifestSchema,
+    );
+
+    const failures: RuleExecutionFailure[] = await jestConform(
+      templateManifest.scripts,
+      projectManifest.scripts,
+    );
+
+    return failures.length === 0 ? pass() : fail(failures);
+  },
+});
+
+/**
+ * Validates whether target project has all the required jest scripts matching with template project.
+ *
+ * @param templateScripts - The record of key and value from template scripts.
+ * @param projectScripts - The record of key and value from project scripts.
+ */
+async function jestConform(
+  templateScripts: Record<string, string>,
+  projectScripts: Record<string, string>,
+): Promise<RuleExecutionFailure[]> {
+  const jestScriptsRequired = ['test', 'test:watch'];
```

Comment:
This rule checks that `test` and `test:watch` are present in the project, but it assumes that they both include `jest` somewhere. I'm wondering whether we should:

- Call this rule `package-test-scripts-conform`, and group this rule together with the more Jest-specific rules later
- Double-check that both the `test` and `test:watch` scripts include "jest"

I'm inclined to go with the first option, but I'm curious to hear your thoughts.